exchange/wrappers/buzzex.js

```diff
@@ -0,0 +1,303 @@
+const Buzzex = require('buzzex-gekko-npm');
+const moment = require('moment');
+const _ = require('lodash');
+const exchangeUtils = require('../exchangeUtils');
+const retry = exchangeUtils.retry;
+const scientificToDecimal = exchangeUtils.scientificToDecimal;
+
+const marketData = require('./buzzex-markets.json');
+
+const Trader = function(config) {
+  _.bindAll(this);
+
+  if(_.isObject(config)) {
+    this.key = config.key;
+    this.secret = config.secret;
+    this.currency = config.currency.toUpperCase()
+    this.asset = config.asset.toUpperCase();
+  }
+
+  this.name = 'Buzzex';
+  this.since = null;
+  
+  this.market = _.find(Trader.getCapabilities().markets, (market) => {
+    return market.pair[0] === this.currency && market.pair[1] === this.asset
+  });
+  this.pair = this.market.book;
+
+  this.interval = 3100;
+
+  this.buzzex = new Buzzex(
+    this.key,
+    this.secret,
+    {timeout: +moment.duration(60, 'seconds')}
+  );
+}
+
+const recoverableErrors = [
+  
+];
+
+// errors that might mean
+// the API call succeeded.
+const unknownResultErrors = [
+  'Response code 502',
+  'Response code 504',
+  'Response code 522',
+  'Response code 520',
+]
+
+const includes = (str, list) => {
+  if(!_.isString(str))
+    return false;
+
+  return _.some(list, item => str.includes(item));
+}
+
+Trader.prototype.handleResponse = function(funcName, callback, nonMutating, payload) {
+  return (error, body) => {
+
+    if(!error && !body) {
+      error = new Error('Empty response');
+    }
+
+    if(error) {
+      if(includes(error.message, recoverableErrors)) {
+        error.notFatal = true;
+      }
+
+      if(includes(error.message, ['Rate limit exceeded'])) {
```

Comment:
question: Is the Buzzex rate limit message exactly the same as the Kraken rate limit message?

Reply:
> Some more nits.
> 
> The issue here is that Kraken's API is very hard to work with, and Gekko has a lot of exception handling code to make sure everything stays working. I don't know the Buzzex API, but I'm not assuming it behaves exactly the same way.
> 
> Please go through the full file and remove everything kraken specific. Don't only fix the lines I commented on... It's better to NOT handle buzzex errors than to leave old Kraken code that handles errors Buzzex never responds with (this way we can easily add error handling for Buzzex).
> 
> Thanks!

Thanks for this. If it's ok to leave error handling at this stage, ill remove codes for it for now. I need to confirm with other devs how's the error messaging of buzzex. Will removing all the codes inside `if(error)` do?